website/content/docs/concepts/iam.mdx

```diff
@@ -0,0 +1,103 @@
+---
+layout: docs
+page_title: Identity and Access Management (IAM)
+description: |-
+  Identity and Access Management in Boundary
+---
+
+# Identity and Access Management (IAM)
+## IAM Components
+There are 5 key components to Boundary's identity and access management system: **Scopes**, **Auth Methods**, **Accounts**, **Users**, **Groups**, and **Roles**.
+
+#### Scopes
+[Scopes](https://developer.hashicorp.com/boundary/docs/concepts/domain-model/scopes) are containers for all of Boundary's resources and have their own permissions 
+boundary, isolated from other scopes at the same level. **Each scope has its own set of resources and permissions**. There are three types of scopes:
+- **Global** - The single, outermost scope of Boundary that cannot be deleted.
+- **Org (Organization)** - Org scopes are contained in the global scope and are typically used to represent business units, organizations, or departments. Org scopes 
```

Comment:
So if I need multiple IdPs (auth-methods) I should put them in separate scopes?
What if one business unit/department needs many IdPs for its cohort?